### PR TITLE
Early vibration multiplier feature implemented

### PR DESCRIPTION
**Pull request by bnm12**
First version of the vibration multiplier feature

### CODE REVIEW

File: Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs [hunk 1]
```diff
@@ -142,14 +155,17 @@ private void Detach()
                     {
                         continue;
                     }
+                    double vibeSpeed = (_lastVibration.LeftMotorSpeed + _lastVibration.RightMotorSpeed) / (2.0 * 65535.0);
                     await _bpServer.SendMessage(new SingleMotorVibrateCmd(device.Index,
-                        (_lastVibration.LeftMotorSpeed + _lastVibration.RightMotorSpeed) / (2.0 * 65535.0)));
+                        vibeSpeed * _vibrationMultiplier > 1 ? 1 : vibeSpeed * _vibrationMultiplier));
```
Comment (qdot) [inline Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs]: Let's simplify this a bit and just make it Math.Min(vibeSpeed * _vibrationMultiplier, 1.0)

File: Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs [hunk 2]
```diff
@@ -70,6 +74,10 @@ public MainWindow()
             ButtplugTab.AddDevicePanel(_bpServer);
             ButtplugTab.SelectedDevicesChanged += OnSelectedDevicesChanged;
 
+            _vibeTab = new VibeConfigTab();
+            _vibeTab.MultiplierChanged += MultiplierChanged;
+            ButtplugTab.SetOtherTab2("Vibe Config", _vibeTab);
```
Comment (qdot) [inline Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs]: Hmm. It kinda feels like we should maybe create a system to push new tabs versus just adding tabs manually like this, but I guess it works for now (and I vaguely remember having some issue trying to implement that when I made the vibration graph tab). 

If I ever get huffy about this in the future, remind me I made this comment.

Comment (bnm12) [reply Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs]: Yeah I was kinda thinking the same but I'm still not that great at WPF and didn't really feel too comfortable trying a major(ish) rewrite when I noticed how the other one was added... :/

File: Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs (hunk 1 above)
Comment (bnm12) [reply Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs]: Totally forgot that Math.Min existed... And now I probably never will again (baring any more 3am coding)

**Pull request closed without merging**